index.js

```diff
@@ -1,11 +1,9 @@
-var buffer = require('buffer')
-
 if (Buffer.from && Buffer.alloc && Buffer.allocUnsafe && Buffer.allocUnsafeSlow) {
-  module.exports = buffer
+  module.exports = Buffer
```

Comment:
This is a huge breaking change. `require('buffer') !== Buffer`. Try doing `require('buffer')` and look at what you get.